first.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+// 1
+
+let numberSetOne = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12];
+
+function modifyNumbers(numbers) {
+    let modifyNumberSet = [];
+
+    for (let i = 0; i < numbers.length; i++) {
+        if (numbers[i] % 2 === 0) {
+            modifyNumberSet.push(numbers[i] += 1);
+        } else {
+            modifyNumberSet.push(numbers[i] -= 1);
```

Comment:
you use `numbers[i]` 2 times. better to extract this into variable and then use.

`const currentNumber = numbers[i]`